keras/ops/math.py

```diff
@@ -310,6 +312,11 @@ def qr(x, mode="reduced"):
            [-0.5070925   0.2760267 ]
            [-0.8451542  -0.34503305]], shape=(3, 2), dtype=float32)
     """
+    warnings.warn(
```

Comment:
Rather than duplicating the code, just remove the code here and add the export alias `ops.linalg.xyz` to the remaining version. No need for a deprecation message, it's fine to have API aliases.

Reply:
I have removed the duplicate code in `math.py` and added the export as `keras.ops.xyz` on the current version in `linalg.py`.